ui/app/src/components/ContentTypeManagement/components/controls/ContentPathInput.tsx

```
@@ -0,0 +1,98 @@
+				autoFocus={autoFocus}
+				id={htmlId}
+				fullWidth
+				inputProps={{ maxLength }}
+				value={value}
+				onChange={handleChange}
+				disabled={readonly}
+				endAdornment={
+					<IconButton
+						aria-label="Maximize"
+						onClick={() => onOpenPathSelectionDialog()}
+						children={<SearchRoundedIcon />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)